Britannia/events/BTWK_storyline_events.txt

```diff
@@ -4,96 +4,676 @@
+				marry = character:19
+			}
+		}
+		character:1	= {
+			trigger_event = BTWK_story_event.0105
+		}
+	}	
+	################### Belgae ###################
+	option = { 
+		name = BTWK_story_event.0104.f
+		trigger = {
+			any_in_global_list = {
+				variable = char_is_norwenna_wedding_prospect
+				OR = {
+					has_primary_title = title:k_belgae
+					AND = {
+						is_primary_heir_of = title:k_belgae.holder
+						holds_any_storyline_britannia_kingdoms = no
+						}
+					}
+				}
+			}
+		if = {
+			limit = { 
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					has_primary_title = title:k_belgae
+				}
+			}
+			title:k_gwent.holder = {
+				marry = character:19
+			}
+		}
+		else_if = {
+			limit = {
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					is_primary_heir_of = title:k_belgae.holder
+				}
+			}
+			title:k_gwent.holder.primary_heir = {
+				marry = character:19
+			}
+		}
+		character:1	= {
+			trigger_event = BTWK_story_event.0105
+		}
+	}	
+	################### Cynwidion ###################
+	option = { 
+		name = BTWK_story_event.0104.g
+		trigger = {
+			any_in_global_list = {
+				variable = char_is_norwenna_wedding_prospect
+				OR = {
+					has_primary_title = title:k_cynwidion
+					AND = {
+						is_primary_heir_of = title:k_cynwidion.holder
+						holds_any_storyline_britannia_kingdoms = no
+						}
+					}
+				}
+			}
+		if = {
+			limit = { 
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					has_primary_title = title:k_cynwidion
+				}
+			}
+			title:k_gwent.holder = {
+				marry = character:19
+			}
+		}
+		else_if = {
+			limit = {
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					is_primary_heir_of = title:k_cynwidion.holder
+				}
+			}
+			title:k_gwent.holder.primary_heir = {
+				marry = character:19
+			}
+		}
+		character:1	= {
+			trigger_event = BTWK_story_event.0105
+		}
+	}	
+	################### Demetia ###################
+	option = { 
+		name = BTWK_story_event.0104.h
+		trigger = {
+			any_in_global_list = {
+				variable = char_is_norwenna_wedding_prospect
+				OR = {
+					has_primary_title = title:k_demetia
+					AND = {
+						is_primary_heir_of = title:k_demetia.holder
+						holds_any_storyline_britannia_kingdoms = no
+						}
+					}
+				}
+			}
+		if = {
+			limit = { 
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					has_primary_title = title:k_demetia
+				}
+			}
+			title:k_gwent.holder = {
+				marry = character:19
+			}
+		}
+		else_if = {
+			limit = {
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					is_primary_heir_of = title:k_demetia.holder
+				}
+			}
+			title:k_gwent.holder.primary_heir = {
+				marry = character:19
+			}
+		}
+		character:1	= {
+			trigger_event = BTWK_story_event.0105
+		}
+	}	
+}
+
+BTWK_story_event.0105	= { # Guardian Selection Event
```

Comment:
Not entirely sure what's going on here - I thought that marrying Norwenna was basically supposed to be cementing the alliance/guardian status, so it doesn't make sense that the suitor and the guardian are separate? Unless this is for the regency council or something?

Reply:
I don't think this was finished when I pushed it. The event was there to represent the characters who pledged allegiance to Uther/Mordred that weren't marrying Norwenna (Arthur, Merlin & Tewdric - I think Gundleus was the 4th character but that's only through marriage). I'd split the marriage event and the guardian event to set the scene for when Uther caves to make Arthur a guardian . I believe that we discussed that the "non-marriage" guardians were irrelevant, so I left them as they were canonically.

Reply:
It has no functionality, and I don't think I intended it to, just purely a "story" event.

Reply:
Ah okay, I got confused when I saw "selection"

Reply:
No worries, I should have been more clear with the comment.